### PR TITLE
Kubernetes changes

### PR DESCRIPTION
**Pull request by gws**
Changes to the template made to support Kubernetes deployment and clean up some unused Fleet / Consul scripts.

Change highlights:

- Tweak `script/build` mainly to tell Buildkite which image got built, and some small changes
- Remove `script/deploy` (Fleet-only, not used anymore)
- Add `.buildkite/` with pipeline YAML

We can release `0.6.0` if this gets merged, but I'm seeing now that @mrichards42 has an outstanding PR, so we might need to coordinate on this a bit!

### CODE REVIEW

File: resources/leiningen/new/kraken_works/script/build [hunk 1]
```diff
@@ -65,4 +71,6 @@ perl -p -i -e "s/^Environment=CONTAINER=.*$/Environment=CONTAINER=${SERVICE}/" $
 if hash buildkite-agent 2>/dev/null ; then
   echo '--- saving service file'
   buildkite-agent artifact upload ${SERVICE}@.service
```
Comment (cap10morgan) [inline resources/leiningen/new/kraken_works/script/build]: We can get rid of the previous two lines as well! 🔪

**Review by cap10morgan — APPROVED** [review]
LGTM with some additional excision I commented on.

**Comment by gws** [comment]
Thanks to @katrinamariehh for pushing on this as we worked through deploying her new service.

File: resources/leiningen/new/kraken_works/script/build (hunk 1 above)
Comment (gws) [reply resources/leiningen/new/kraken_works/script/build]: :hocho: :+1:

Comment (gws) [reply resources/leiningen/new/kraken_works/script/build]: Actually looks like a bunch of service template stuff can get removed... another commit incoming.

File: project.clj [hunk 2]
```diff
@@ -1,4 +1,4 @@
-(defproject kraken-works/lein-template "0.5.3-SNAPSHOT"
+(defproject kraken-works/lein-template "0.6.1-SNAPSHOT"
```
Comment (tie-rack) [inline project.clj]: Not a big deal _at all_, but typically PRs should leave the version alone. `lein release` handles all that. With this change, `lein release` would create and push 0.6.1, which isn't next.

Comment (gws) [reply project.clj]: @tie-rack Sorry, I do understand this but I've done releases manually for too long. I will remove these commits.

Comment (gws) [reply project.clj]: @tie-rack If the next release *wouldn't* be `0.5.3` (just hypothetically), then the PR should bump the `-SNAPSHOT` version though (to the version that would be released without the `-SNAPSHOT` tag), right?

**Review by mrichards42 — APPROVED** [review]
The two main changes in that PR aren't very big, so we could combine them in this one:

1. Fix the `DATOMIC_URI` -- a220c9e if you want to cherry-pick
2. Updating dependency versions, which since that PR is old, might be out of date by now.

If we're going to release a new template it would probably be good to at least have the deps up to date

File: project.clj (hunk 2 above)
Comment (tie-rack) [reply project.clj]: @gws If the next one should be `0.6.0`, then we'll run `lein release :minor`. No manual editing of the version number required.

**Comment by gws** [comment]
@mrichards42 Do you want to refresh deps and merge your approved PR and I can rebase this one on new `master`? That feels a little nicer to me, but you might feel differently...

**Pull request merged**